src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -428,19 +452,34 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       symmetric: {
         inputType: 'options',
         required: false,
-        options: [
-          { name: 'False', value: false },
-          { name: 'True', value: true },
-        ],
+        options: booleanOptions,
```

Comment:
Oh interesting, this is probably better, but did you find [the "boolean" arg inputType](https://github.com/KittyCAD/modeling-app/blob/main/src/lib/settings/initialSettings.tsx#L462-L464) not workable for these?

Reply:
WOAH https://github.com/KittyCAD/modeling-app/commit/5341e1a5c79f5be31d3fc03a272337e17962dd67

Reply:
all good to me I updated the tests to match